twisted/words/xish/xmlstream.py

```diff
@@ -14,6 +14,20 @@
 usable. See Twisted Words for Jabber specific protocol support.
 
 Maintainer: Ralph Meijer
+
+@var STREAM_CONNECTED_EVENT: Token indicating the connection has been
+    established.
+@type STREAM_CONNECTED_EVENT: L{intern}.
+
+@var STREAM_END_EVENT: Token indicating the connection has been closed.
+@type STREAM_CONNECTED_EVENT: L{intern}.
+
+@var STREAM_ERROR_EVENT: Token indicating a parse error occurred.
+@type STREAM_ERROR_EVENT: L{intern}.
+
+@var STREAM_START_EVENT: Token indicating the parsing of the document has
+    begin.
```

Comment:
… has *begun*? Also, more precisely, this event signals that the root element of the XML Stream has been received. For XMPP, this would be the `<stream:stream …>` opening tag.